backend/src/v5/utils/responseCodes.js

```diff
@@ -66,6 +66,8 @@ ResponseCodes.templates = {
 
 	// Drawing related error
 	drawingNotFound: { message: 'Drawing not found.', status: 404 },
+	calibrationNotFound: { message: 'Calibration not found.', status: 404 },
+	revisionNotUnconfirmed: { message: 'Revision is not is an unconfirmed state.', status: 400 },
```

Comment:
actually maybe we should just use calibrationNotFound instead of having a different error.

The action is youo're trying to confirm a calibration, and there is none so calibration not found makes sense.